android/src/main/java/com/reactnativecommunity/netinfo/ConnectivityReceiver.java

```diff
@@ -37,7 +37,6 @@ public abstract class ConnectivityReceiver {
     private final WifiManager mWifiManager;
     private final TelephonyManager mTelephonyManager;
     private final ReactApplicationContext mReactContext;
-    public boolean hasListener = false;
```

Comment:
This feature was added to address privacy-related store rejections, in #548
How do you plan to defer getting the MAC address until the user has consented, or is this no longer necessary specifically because of the turbomodule migration (and if so, how exactly - please be specific so we make sure not to blow up approval process for all Chinese apps that use this module)

Reply:
wow. Thanks for pointing this out. Didn't know this. I thought it was for performance reasons. 
I've started a [discussion](https://github.com/reactwg/react-native-new-architecture/discussions/74) regarding some changes in NativeEventEmitter. The addListener and removeListeners methods doesn't seem to be triggering on Android with Turbomodule. I'll verify again.

Reply:
Great - hopefully there is resolution on that one - much of the netinfo information may be considered an information leak so perhaps the restrictions won't stop at MAC address only, and this was the best way (we thought!) to do a "privacy related total information cutout" so it either must be handled or there must be some other way to defer calling any platform APIs until desired

Reply:
@mikehardy I just tried and those methods are working on fabric and non-fabric. I think I messed up something while testing! I have added them back, so should be fine now. Thanks again!